synapse/tests/test_cortex.py

```diff
@@ -1,26 +1,22 @@
 import synapse.exc as s_exc
 import synapse.common as s_common
+import synapse.tests.common as s_t_common
```

Comment:
We've been doing this as `import synapse.tests.common as s_test` in other code

Reply:
Fixing :-)